File: blinkeye-framewk-aprobaciones-dominio/src/main/java/net/royal/spring/workflow/dominio/dto/DtoSelectorDinamico.java
package net.royal.spring.workflow.dominio.dto;

import net.royal.spring.framework.modelo.generico.DominioPaginacion;
import net.royal.spring.framework.modelo.generico.dto.DtoTabla;

public class DtoSelectorDinamico extends DtoTabla {
	private DominioPaginacion paginacion;

	public DtoSelectorDinamico() {
		this.paginacion = new DominioPaginacion();
	}

	public DominioPaginacion getPaginacion() {
		return paginacion;
	}

	public void setPaginacion(DominioPaginacion paginacion) {
		this.paginacion = paginacion;
	}

}

File: web-deteccion/src/app/royal/modulos/framework-workflow/dominio/dto/DtoSelectorDinamico.ts
import { DtoTabla } from './../../../../framework/modelo/generico/dto/DtoTabla';
import { DominioPaginacion } from './../../../../framework/modelo/generico/DominioPaginacion';

export class DtoSelectorDinamico extends DtoTabla {
    paginacion: DominioPaginacion;
    constructor() {
        super();
        this.paginacion = new DominioPaginacion();
    }
}